code/WorkInProgress/rewardsLocker.dm

```diff
@@ -283,6 +283,38 @@
 		new /obj/item/pod/paintjob/tronthing(get_turf(activator))
 		return 1
 
+/datum/achievementReward/respirator
+	title = "(Skin) Gas Respirator"
+	desc = "Replaces a gas mask you're holding with a gas respirator. Use inhand to change style."
```

Comment:
nit: Why holding? The SWAT gas mask uses wearing code. These two should probably be consistent as they touch the same clothing piece.

Reply:
fixed (hopefully)